File: Server/src/main/java/jk_5/nailed/server/command/CommandLobby.java
package jk_5.nailed.server.command;

import jk_5.nailed.api.*;
import jk_5.nailed.api.map.*;
import jk_5.nailed.api.player.*;

/**
 * No description given
 *
 * @author jk-5
 */
public class CommandLobby extends NailedCommand {

    public CommandLobby() {
        super("lobby");
    }

    @Override
    public void processCommandPlayer(Player sender, Map map, String[] args) {
        sender.teleportToMap(NailedAPI.getMapLoader().getLobby());
    }
}

File: Server/src/main/scala/jk_5/nailed/server/command/CommandLobby.scala
package jk_5.nailed.server.command

import jk_5.nailed.api.player.Player
import jk_5.nailed.api.map.Map

/**
 * No description given
 *
 * @author jk-5
 */
object CommandLobby extends ScalaCommand {

  val name = "lobby"
  val usage = "/lobby - Teleports you to the lobby"

  override def processCommandPlayer(sender: Player, map: Map, args: Array[String]){
    sender.teleportToLobby()
  }
}
